src/test/java/teammates/test/cases/ui/AdminSearchPageActionTest.java

```diff
@@ -12,8 +12,8 @@ public static void classSetUp() {
     
     @Test
     public void testExecuteAndPostProcess() {
-        // TODO : Test is done as a browser test,
-        // cuz gaeSimulation gives problems when rebuilding the document
+        //Test is done as a browser test,
+		//because otherwise there are problems when rebuilding the document
```

Comment:
@NWuensche you should not use tab characters to introduce indentation; use 4 spaces instead. Also you may want to add a space right after `//` just like how it was before.